installer/README.md

```diff
@@ -458,6 +458,22 @@ Kubernetes objects, such as your TLS certificate or connection secrets.
 kubectl create namespace gitpod
 ```
 
+# I need to specify the load balancer - what do I do?
+
+By default, this will create a `proxy` service with `ServiceType: LoadBalancer`.
+If you need to specify your own load balancer, you can do this easily by changing
+your Installer configuration:
+
+```yaml
+meta:
```

Comment:
Unlike `workspace` - which refers workspaces rather than the team - the term `meta` does not make sense from a user's perspective.

Reply:
Fair point. Have you got a preferred name?